### PR TITLE
Misc Docs Edits

### LINKED ISSUE #3725: Add informational block about a precedence for connection storages in Airflow
### Issue Type

Missing content for an existing feature

### Links to Affected Docs

https://docs.astronomer.io/astro/authorize-deployments-to-your-cloud?tab=aws#step-2-create-an-airflow-connection

### The Issue or Context

Customers can use same names for connections in different places. As result connection can be overwritten based on a precedence and customers will spend extra time for debugging.

### Fix or Proposed Change

Following snippet could inform customer about potential issues. It can be organized as Informational block.

```
If you use a mix of strategies for managing connections, it's important to understand that if the same connection is defined in multiple ways, Airflow uses the following order or precedence:

Secrets Backend
Astro Environment Manager
Environment Variables
Airflow's metadata database (Airflow UI)
```

### Additional Notes

_No response_

### Required Reviewers

_No response_

### LINKED ISSUE #3886: Unzip should be put in packages.txt
### Issue Type

Inaccurate, misleading, or out-of-date information

### Links to Affected Docs

https://www.astronomer.io/docs/astro/kubernetespodoperator#:~:text=Add%20the%20unzip%20package%20to%20your%20requirements.txt%20file%20to%20make%20the%20unzip%20command%20available%20in%20your%20Docker%20container%3A

### The Issue or Context

Working with a customer to set up running a KPO on an external cluster.  We hit an issue with one of the steps.  We say to add `unzip` to the `requirements.txt` file.  We kept having issue on the deploy, with the error `unzip module not found`.  We added it to the `packages.txt` instead and it worked.

### Fix or Proposed Change

Change the file you add `unzip` to.  It should be added to the `packages.txt` not the `requirements.txt`

### Additional Notes

_No response_

### Required Reviewers

_No response_

### LINKED ISSUE #3905: Slim Distributions are missing from docs after refactor
### Issue Type

Missing content for an existing feature

### Links to Affected Docs

- https://www.astronomer.io/docs/software/runtime-image-architecture#distribution
- https://www.astronomer.io/docs/astro/runtime-image-architecture#distribution

### The Issue or Context

Return https://www.astronomer.io/docs/software/0.34/runtime-image-architecture#slim-images to 0.35 and Astro Runtime image architecture docs

### Fix or Proposed Change

Re-add slim image info after refactor removed these sections: https://github.com/astronomer/docs/pull/3699

### Additional Notes

https://astronomer.slack.com/archives/C015V2JFKT5/p1720481891084679

### Required Reviewers

hatter

### PR DESCRIPTION
**Pull request by lzdanski**
Consolidating miscellaneous small docs changes into one PR for simplicity. 

Resolves #3725 
Resolves #3905 
Resolves #3886

### CODE REVIEW

File: astro/authorize-deployments-to-your-cloud.md [hunk 1]
```diff
@@ -93,6 +93,17 @@ Now that your Deployment is authorized, you can connect it to your cloud using a
 
 If you don't see **Amazon Web Services** as a connection type in the Airflow UI, ensure you have installed its provider package in your Astro project's `requirements.txt` file. See **Use Provider** in the [Astronomer Registry](https://registry.astronomer.io/providers/Amazon/versions/latest) for the latest package.
```
Comment (lzdanski) [inline astro/authorize-deployments-to-your-cloud.md:94]: @afastronomer - changes related to issue you opened https://github.com/astronomer/docs/issues/3725

File: astro/runtime-image-architecture.mdx [hunk 2]
```diff
@@ -148,30 +148,79 @@ In Airflow, the executor is responsible for determining how and where a task is
```
Comment (lzdanski) [inline astro/runtime-image-architecture.mdx:148]: @RNHTTR - Changes related to issue you reported: https://github.com/astronomer/docs/issues/3905

**Review by afastronomer — APPROVED** [review]

File: astro/kubernetespodoperator.md [hunk 3]
```diff
@@ -444,7 +444,7 @@ To connect to your external EKS cluster, you need to install the AWS CLI in your
     USER astro
     ```
 
-2. Add the `unzip` package to your `requirements.txt` file to make the `unzip` command available in your Docker container:
+2. Add the `unzip` package to your `packages.txt` file to make the `unzip` command available in your Docker container:
```
Comment (lzdanski) [inline astro/kubernetespodoperator.md:447]: Addressed https://github.com/astronomer/docs/issues/3886

File: astro/authorize-deployments-to-your-cloud.md [hunk 4]
```diff
@@ -154,6 +165,17 @@ Now that your Deployment is authorized, you can connect it to your cloud using a
...
 
+:::tip
+
+If you use a mix of strategies for managing connections, if you define the same connection in multiple ways, Airflow uses the following order or precedence:
```
Comment (TJaniF) [inline astro/authorize-deployments-to-your-cloud.md:170]:
```suggestion
If you use a mix of strategies for managing connections, if you define the same connection in multiple ways, Airflow uses the following order of precedence:
```

File: astro/authorize-deployments-to-your-cloud.md [hunk 5]
```diff
@@ -238,6 +260,17 @@ Now that your Deployment is authorized, you can connect it to your cloud using a
 
 If you don't see **Google Cloud** as a connection type in the Airflow UI, ensure you have installed its provider package in your Astro project's `requirements.txt` file. See **Use Provider** in the [Astronomer Registry](https://registry.astronomer.io/providers/Google/versions/latest) for the latest package.
 
+:::tip
+
+If you use a mix of strategies for managing connections, if you define the same connection in multiple ways, Airflow uses the following order or precedence:
```
Comment (TJaniF) [inline astro/authorize-deployments-to-your-cloud.md:265]:
```suggestion
If you use a mix of strategies for managing connections, if you define the same connection in multiple ways, Airflow uses the following order of precedence:
```

**Review by TJaniF — APPROVED** [review]
LGTM :) 
Just two nit suggestions. I did not know you could combine image tags! TIL

**Pull request merged**